java_tests/src/main/java/org/infinispan/client/hotrod/configuration/ServerConfigurationBuilder.java

```diff
@@ -8,11 +8,10 @@
  * @author Tristan Tarrant
  * @since 5.3
  */
-public class ServerConfigurationBuilder extends AbstractConfigurationChildBuilder implements Builder<ServerConfiguration> {
+public class ServerConfigurationBuilder {
```

Comment:
The AbstractConfigurationChildBuilder and IBuilder are not used anywhere right now so they should be removed as long as these changes are acceptable. But I think it's not possible (see my previous comment).